### DISCUSSION TITLE
Using modelScan on Zephyr-7b-beta.Q4_K_S.gguf

### DISCUSSION THREAD

**Discussion by frederickrohn** [protectai/modelscan#177] [Q&A]
Hi, I downloaded Zephyr-7b-beta.Q4_K_S.gguf off of the huggingface website, and I wanted to test it for vulnerabilities. When I ran modelscan -p zephyr-7b-beta.Q4_K_S.gguf --show-skipped, it shows that the model was not scanned because "not settings file detected". Yes, there isn't a settings file, I just downloaded the model, it's not a repository. How do I make this work?

**Comment by TFWol** [comment]
I was wondering something similar, but I found it's not supported: https://github.com/protectai/modelscan/issues/111
